walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -24,7 +24,11 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
```

Comment:
It seems that this serialVersionUID is not used? Prefer not to put it in the code.

Reply:
It was just giving me a warning otherwise, but I'll take it out.